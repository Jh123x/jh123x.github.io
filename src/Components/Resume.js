import React from "react";
import styled from "styled-components";
import { InnerLayout } from "../styles/Layouts";
import Title from "../Components/Title";
import SmallTitle from "../Components/SmallTitle";
import {
  BusinessCenter as BusinessCenterIcon,
  School as SchoolIcon,
} from "@mui/icons-material";
import ResumeItem from "../Components/ResumeItem";
import workExperience from "../data/workExperience";
import schExperience from "../data/schExperience";

const Resume = () => {
  const briefcase = <BusinessCenterIcon />;
  const school = <SchoolIcon />;
  return (
    <ResumeStyled>
      <Title title={"Resume"} span={"resume"} />
      <InnerLayout>
        <div className="small-title">
          <SmallTitle icon={briefcase} title={"Working Experience"} />
        </div>
        <div className="resume-content">
          {workExperience.map((experience) => <ResumeItem key={experience.title + experience.year} {...experience}/>)}
        </div>
        <div className="small-title u-small-title-margin">
          <SmallTitle icon={school} title={"Educational Qualifications"} />
        </div>
        <div className="resume-content ">
          {schExperience.map((experience) => <ResumeItem key={experience.title + experience.year} {...experience}/>)}
        </div>
      </InnerLayout>
    </ResumeStyled>
  );
};

const ResumeStyled = styled.section`
  .small-title {
    padding-bottom: 3rem;
  }
  .u-small-title-margin {
    margin-top: 4rem;
  }

  .resume-content {
    border-left: 2px solid var(--border-color);
  }
`;
export default Resume;
