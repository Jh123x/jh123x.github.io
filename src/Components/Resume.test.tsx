import * as React from "react";
import { describe, expect, test } from "@jest/globals";
import Resume from "./Resume";

describe("Resume", () => {
  test("should render correctly", () => {
    expect(<Resume />).toMatchSnapshot();
  });
});
