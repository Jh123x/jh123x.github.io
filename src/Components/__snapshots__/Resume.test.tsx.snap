// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Resume should render correctly 1`] = `<Resume />`;
